csharp/examples/FlightAspServerExample/appsettings.Development.json

```diff
@@ -0,0 +1,8 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
```

Comment:
These settings are the same as `appsettings.json`. Can we just remove the `Development` file?